README.md

```diff
@@ -226,11 +225,11 @@ Alternatively, you can block until the rate limit is reset by using the `context
 repos, _, err := client.Repositories.List(context.WithValue(ctx, github.SleepUntilPrimaryRateLimitResetWhenRateLimited, true), "", nil)
 ```
 
-You can use [go-github-ratelimit](https://github.com/gofri/go-github-ratelimit) to handle
+You can use [gofri/go-github-ratelimit](https://github.com/gofri/go-github-ratelimit) to handle
 secondary rate limit sleep-and-retry for you.
 
-Learn more about GitHub secondary rate limiting at
-https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#about-secondary-rate-limits .
+Learn more about GitHub secondary rate limiting in the
+["About secondary rate limits"](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#about-secondary-rate-limits).
```

Comment:
@Parker77 - what would you recommend for this sentence for proper grammar?
I'm leaning toward `in "About secondary rate limits"` over "at" or "in the".
Thoughts?

Reply:
I would agree, "in" (with no "the") would be preferable. Same on line 208 above.